python/lsst/dax/imgserv/locateImage.py

```diff
@@ -450,9 +470,38 @@ def _cutoutBoxPixels(srcImage, xyCenter, width, height, log):
     imgSub = srcImage[pixULX:pixEndX, pixULY:pixEndY].clone()
     return imgSub
 
+
+def _getKeysFromList(flist, fields):
+    '''flist assumed to be dictionary
+    '''
+    vals = []
+    for f in fields:
+        for l in flist:
+            if f == l:
+                vals.append(flist.get(f))
+    return vals
```

Comment:
'l' should not be used for a variable name. It looks too much like '1'.

Would this work?
```
for f in fields:
    vals.append(flist.get(f))
```